cogs/events.py

```diff
@@ -1,33 +1,36 @@
 import traceback
 
-import discord
-from discord.ext import commands
+from discord.ext import Bot
 
+from discrod import Color, Embed, Message
+from discrod.ext.commands import Cog
 
-class Custom(commands.Cog):
 
-    def __init__(self, client):
-        self.client = client
+class Custom(Cog):
 
-    @commands.Cog.listener()
-    async def on_message(self, message):
-        if message.author.id == self.client.user.id:
+    def __init__(self, bot: Bot) -> None:
+        bot.bot = bot
+
+    @Cog.listener()
+    async def on_message(self, message: Message) -> None:
+        # Check that the message is not from bot account
+        if message.author.bot:
             return
 
         if message.content.lower().startswith("help"):
             await message.channel.send("Hey! Why don't you run the help command with `>>help`")
 
         await self.client.process_commands(message)
```

Comment:
This part is updated by me in **update-events**

Reply:
Should be fine when it will get merged to master it will get resolved. I'm adding type hints here, which is the most important change here, it shouldn't be a problem.